src/main/java/ch/njol/skript/effects/EffLog.java

```diff
@@ -50,20 +50,20 @@
 @Description({"Writes text into a .log file. Skript will write these files to /plugins/Skript/logs.",
 		"NB: Using 'server.log' as the log file will write to the default server log. Omitting the log file altogether will log the message as '[Skript] [&lt;script&gt;.sk] &lt;message&gt;' in the server log."})
 @Examples({"on place of TNT:",
-		"   log \"%player% placed TNT in %world% at %location of block%\"",
-		"	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"",
-		"	log \"%player% placed TNT in %world% at %location of block%\" to file \"tnt/placement.log\"\"with a severity of warning\""})
+		"log \"%player% placed TNT in %world% at %location of block%\"",
```

Comment:
Why just TNT? Also these are still basically the same examples, just in a different sentence order and with more or less information

Reply:
because it's on place of tnt

Reply:
![image](https://github.com/SkriptLang/Skript/assets/66171067/80dc81ac-180e-462d-9f0e-52e6224da561)